src/cleanvision/issue_managers/image_property_issue_manager.py

```diff
@@ -162,49 +158,100 @@ def find_issues(
 
-            scores = self.image_properties[issue_type].get_scores(
-                raw_scores=property_values, **self.params[issue_type]
+            scores[issue_type] = self.image_properties[issue_type].get_scores(
+                raw_scores=values, **self.params[issue_type]
             )
+        return scores
 
-            # Update issues
-            self.issues[f"{issue_type}_score"] = scores
-            self.issues[get_is_issue_colname(issue_type)] = self.image_properties[
-                issue_type
-            ].mark_issue(scores, self.params[issue_type].get("threshold"))
-
-            summary_dict[issue_type] = self._compute_summary(
-                self.issues[get_is_issue_colname(issue_type)]
+    def aggregate_comp(
+        self, results: List[Dict[str, Any]], issue_types: List[str]
+    ) -> Dict[str, pd.DataFrame]:
+        agg_computations: Dict[str, List[Dict[str, Any]]] = {
+            issue_type: [] for issue_type in issue_types
+        }
+        paths = []
+        for result in results:
+            paths.append(result["path"])
+            for issue_type in issue_types:
+                agg_computations[issue_type].append(result[issue_type])
+        for issue_type in issue_types:
+            agg_computations[issue_type] = pd.DataFrame(
+                agg_computations[issue_type], index=paths
             )
+        return agg_computations
 
-        # update issues and summary
-        self.update_summary(summary_dict)
-        return
+    def update_info(
+        self, agg_computations: Dict[str, pd.DataFrame], imagelab_info: Dict[str, Any]
+    ) -> None:
+        for issue_type in self.issue_types:
+            property_name = self.image_properties[issue_type].name
+            if issue_type in agg_computations:
+                comp = agg_computations[issue_type]
+                self.info[issue_type] = {}
 
-    def update_info(self, raw_scores: Dict[str, Any]) -> None:
-        for issue_type, scores in raw_scores.items():
-            # todo: add a way to update info for image properties which are not stats
-            if self.image_properties[issue_type].name is not None:
-                self.info["statistics"][self.image_properties[issue_type].name] = scores
+                for column_name in comp.columns:
+                    if column_name == property_name:
+                        self.info["statistics"][property_name] = comp[property_name]
+                    else:
+                        self.info[issue_type][column_name] = comp[column_name]
+            else:
+                if property_name not in self.info["statistics"]:
+                    self.info["statistics"][property_name] = imagelab_info[
+                        "statistics"
+                    ][property_name]
+                self.info[issue_type] = imagelab_info.get(issue_type, {})
+
+        # todo: revisit when there are more issues using same properties like light and dark
```

Comment:
Is the intention to map compatible issues to each other based on the shared property? 

Suppose an issue (e.g. LIGHT) is valid (exists in `self.issue_types`) but is not set in `self.info`. Do we assume its counterpart (e.g. DARK) has been set with the desired property?

Reply:
Yes. In `imagelab.find_issues` I get a defer set of issues to re-use computations. This defer set will add, all but one of the compatible issues to the defer set. These left out issues will be populated using the one that was left out of defer set.